lucid/lucid-htmx.cabal

```diff
@@ -1,57 +1,68 @@
-cabal-version: 1.12
-
--- This file has been generated from package.yaml by hpack version 0.35.1.
---
--- see: https://github.com/sol/hpack
-
-name:           lucid-htmx
-version:        0.1.0.7
-synopsis:       Use htmx in your lucid templates
-description:    Please see the README on GitHub at <https://github.com/monadicsystems/lucid-htmx#readme>
-category:       Web, HTML
-homepage:       https://github.com/monadicsystems/lucid-htmx#readme
-bug-reports:    https://github.com/monadicsystems/lucid-htmx/issues
-author:         Monadic Systems LLC
-maintainer:     tech@monadic.systems
-copyright:      2022 Monadic Systems LLC
-license:        BSD3
-license-file:   LICENSE
-build-type:     Simple
+cabal-version:      3.6
+name:               lucid-htmx
+version:            0.1.0.7
+synopsis:           Use htmx in your lucid templates
+description:
+  Please see the README on GitHub at <https://github.com/monadicsystems/lucid-htmx#readme>
+
+category:           Web, HTML
+homepage:           https://github.com/monadicsystems/lucid-htmx#readme
+bug-reports:        https://github.com/monadicsystems/lucid-htmx/issues
+author:             Monadic Systems LLC
+maintainer:         tech@monadic.systems
+copyright:          2022 Monadic Systems LLC
+license:            BSD-3-Clause
+license-file:       LICENSE
+build-type:         Simple
 extra-source-files:
-    README.md
-    ChangeLog.md
+  ChangeLog.md
+  README.md
 
 source-repository head
-  type: git
+  type:     git
   location: https://github.com/monadicsystems/lucid-htmx
 
+common def-exts
+  default-extensions:
+    LambdaCase
+    OverloadedStrings
+
 library
+  import:           def-exts
+
+  -- cabal-fmt: expand src
```

Comment:
This magic comment makes it so that `cabal-fmt -i lucid-htmx.cabal` auto populates the `exposed-modules` field, kinda like hpack.